src/Kirel.Repositories/Infrastructure/Generics/GenericRepository.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Linq.Expressions;
```

Comment:
Replace file to root of project and rename to KirelGenericEntityFrameworkRepository

Reply:
Replaced